Geometry_Engine/Modify/RemoveLeastSignificantVertices.cs

```diff
@@ -41,20 +41,14 @@ public static partial class Modify
         [Input("angleTolerance", "The tolerance of the angle that defines a straight line. Default is set to the value defined by BH.oM.Geometry.Tolerance.Angle")]
         [Output("polyline", "The cleaned polyline")]
         public static Polyline RemoveLeastSignificantVertices(this Polyline polyline, double angleTolerance = Tolerance.Angle)
-        {
-            //This method is for closed polylines only at the moment
-            if (!polyline.IsClosed())
-            {
-                BH.Engine.Reflection.Compute.RecordError("The RemoveLeastSignificantVertices method is only for closed polylines and the input polyline is not closed.");
-                return polyline;
-            }
-
+        {            
             List<Point> pnts = polyline.DiscontinuityPoints();
+            List<Point> originalPnts = polyline.DiscontinuityPoints();
```

Comment:
The list is useless (plus duplicate of the above) - it could be `Point lastPoint = pnts.Last();`